Firestore/Swift/Source/SwiftAPI/Pipeline/Aggregation/AggregateFunction.swift

```diff
@@ -0,0 +1,34 @@
+// Copyright 2025 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+public class AggregateFunction: AggregateBridgeWrapper, @unchecked Sendable {
+  var bridge: AggregateFunctionBridge
```

Comment:
Why is this mutable?

Reply:
Thank you for pointing this out! I have change these to non-mutating.